pkg/image/registry/imagestream/strategy.go

```diff
@@ -487,6 +488,79 @@ func (s Strategy) Decorate(obj runtime.Object) error {
 	return nil
 }
 
+// Export prepares the object for exporting.
+func (s Strategy) Export(obj runtime.Object, exact bool) error {
```

Comment:
Ideally, `oc export` should be configured via the resource builder to export the resource (hence indirectly use this method). I will look into it but eventually as per @smarterclayton, the command should have more control on the exported resource (maybe a different PR).